examples/05_tower_monopile/analysis_options.yaml

```diff
@@ -0,0 +1,62 @@
+general:
```

Comment:
This appears to come from the WISDEM examples again, like the previous 03 example clash